lib/vrl/web-playground/README.md

```diff
@@ -0,0 +1,67 @@
+# VRL WASM Web Playground
+
+This directory houses the exposed VRL function to WASM `run_vrl()` used to
+power [Vector Remap Language Playground][vrl-playground], or **VRL Playground**
+for short. Although there is already a local REPL supported for use within the
+terminal, this playground will support running VRL in the web browser and test
+input via uploading event files, or specifying an event via a text input field.
+
+## Setup
+
+To build the project we need to use `wasm-pack`. This compiles our Rust code
+to WebAssembly which can then be used within the browser. Install it by running:
+
+```shell
+cargo install 0.10.3 wasm-pack
+```
+
+After installing `web-pack` we must compile our project by running:
+
+```shell
+wasm-pack build --target web
+```
+
+Notice a `pkg` directory was created which contains `wasm_bg.wasm`, `wasm.js`,
+`wasm.d.ts` these are the files that will be used by the web browser to run the
+compiled Rust code.
+
+For more information on Rust and WebAssembly please visit
+[the mozilla docs][mozilla-wasm-rust-docs] or
+[the Rust book wasm chapter][rust-book-wasm]
+
+The `lib.rs` file is available to the web browser. This will make it so we can
```

Comment:
Maybe re-word this slightly. There is nothing really special about `lib.rs` other than it's the entry point of the `web-playground` library. That's just the file that happens to export the `run_vrl` function. You could theoretically export a function from any file.